back/app/models/project.rb

```diff
@@ -24,6 +24,7 @@
 #  preview_token                :string           not null
 #  header_bg_alt_text_multiloc  :jsonb
 #  hidden                       :boolean          default(FALSE), not null
+#  unlisted                     :boolean          default(FALSE), not null
```

Comment:
Slightly bothering me that we have a double negative here - ie unlisted = false, I do wonder if it's worth flipping it and having the property called `listed` and defaulting it to `true`?

Reply:
Yeah was also considering this, I went with `unlisted` because it is similar to `hidden`. But I will change it to `listed`